src/x3f_process.c

```diff
@@ -68,14 +68,20 @@ static int get_black_level(x3f_t *x3f,
   uint64_t *black_sum;
   double *black_sum_sqdev;
   int pixels, i;
+  /* Workaround for bug in DP2 firmware. DarkShieldBottom is specified
+     incorrectly and thus ingnored. */
+  char *cammodel;
+  int use_bottom = !x3f_get_prop_entry(x3f, "CAMMODEL", &cammodel) ||
+    strcmp(cammodel, "SIGMA DP2");
```

Comment:
The PROP metadata is mostly uppercase. I would expect it to be
consistent.

On sáb, 2015-07-25 at 07:14 -0700, mmroden wrote:
> In src/x3f_process.c:
> 
> > @@ -68,14 +68,20 @@ static int get_black_level(x3f_t *x3f,
> >    uint64_t *black_sum;
> >    double *black_sum_sqdev;
> >    int pixels, i;
> > +  /* Workaround for bug in DP2 firmware. DarkShieldBottom is specified
> > +     incorrectly and thus ingnored. */
> > +  char *cammodel;
> > +  int use_bottom = !x3f_get_prop_entry(x3f, "CAMMODEL", &cammodel) ||
> > +    strcmp(cammodel, "SIGMA DP2");
> 
> stupid question-- there's no potential capitalization issues, ie,
> across different firmware versions?
> 
> —
> Reply to this email directly or view it on GitHub.
> 
>